### ISSUE TITLE
Switch to YouTube Music by Default for Countries that Don't Support GPM

### ISSUE THREAD

**Issue by samantha-cruz** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#3672]
**Checklist:**

* [X ] I have checked that there are no issues with similar or the same content

  **YOU SHOULD CHECK CLOSED ISSUES ASWELL**
* [X ] I have checked the FAQ (https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/wiki/FAQ) and the answer I am looking for is not there
* [ X] I have double checked and can reproduce the issue

**OS:**
Ubuntu 19.10

**GPMDP Version:**
9.7.0

**Issue Descriptions:**
I'm from a country where GPM isn't available, but YTM is, so I can't access anything. I would like to be able to open YTM by default for the first time instead of GPM,

**Steps to Reproduce:**
Install GPMDP in a country where GPM isn't supported.
[debug_info.zip](https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/files/3829680/debug_info.zip)

**Comment by syurchen** [comment]
I second this issue. I have the app switched to YTM as well.

**Comment by tanchekwei** [comment]
Yes, please. I stuck at [this page](https://i.imgur.com/m0mBbqz.png) because Google Music is not available in my country.

**Comment by jostrander** [comment]
Not sure how feasible this is. I'm the mean time you should still be able to right click the tray icon and switch to YTM.
